plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -0,0 +1,86 @@
+package groundwork
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please replace this by
```suggestion
	"github.com/stretchr/testify/require"
```

Reply:
Done